### ISSUE TITLE
Decimation

### ISSUE THREAD

**Issue by ennvina** [ennvina/spellactivationoverlay#50]
The Warlock talent Decimation would be a good candidate to study for a new Spell Alert.

Talents:
- rank 1 https://www.wowhead.com/wotlk/spell=63156/decimation
- rank 2 https://www.wowhead.com/wotlk/spell=63158/decimation

Buff:
- rank 1 https://www.wowhead.com/wotlk/spell=63165/decimation
- rank 2 https://www.wowhead.com/wotlk/spell=63167/decimation

The texture could be displayed on top, using "impact" (texture file already included in addon files)

The proc would light up the Soul Fire buttons
- rank 1 https://www.wowhead.com/wotlk/spell=6353/soul-fire

Questions:
- see if it makes sense to display a Spell Alert
   - below 35% hp, Decimation buffs Soul Fire, which itself proc Decimation
   - basically we end up in a loop of perpetual cast of Soul Fire
   - is it really worth a Spell Alert in this case?
- see if it interferes with other textures displayed on top, namely Molten Core and Backlash
   - top texture of Molten Core is disposable, if needed
   - top texture of Backlash is non-negotiable because it is an official Spell Alert from Retail

**Comment by ennvina** [comment]
Implemented in #68

For more information, please read discussion at https://discord.com/channels/1013194771969355858/1013199984084127794/1019536765155029022
